src/screens/SignUpScreen.tsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
+import ThirdPartyButton from '../components/ThirdPartyButton';
 
 const SignUpScreen = () => {
-  return <div>SignUpScreen</div>;
+  return (
+    <ThirdPartyButton
+      imgSrc="./assets/google-icon.svg"
+      handleClick={() => {}}
+      width={40}
+      height={40}
```

Comment:
Potential enhancement for your button: Adding a "size" property of "S", "M" or "L" instead of hardcoding width and height so the buttons can be predictable sizes